R/conjunct_test_linter.R

```diff
@@ -44,26 +47,37 @@
 conjunct_test_linter <- function(allow_named_stopifnot = TRUE) {
```

Comment:
`assume_dpylr_loaded = FALSE` or a dynamic check if the global `filter()` is from dplyr to remove the NS prefix condition in the XPath?
I think both approaches would greatly increase the amount of true positives.

Reply:
Let's assume `filter()` is `dplyr::filter()` unless another namespace is provided for this initial PR, and fine-tune in follow-up

Reply:
All right